reference/5.1/Microsoft.PowerShell.Utility/Send-MailMessage.md

```diff
@@ -154,6 +154,16 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+[!NOTE]
```

Comment:
I believe you should add `> ` at the beginning of each line in a note

Reply:
@robdy is correct.  See the [Contributor Guide - Markdown Specifics - Notes](https://github.com/PowerShell/PowerShell-Docs/blob/staging/contributing/4-MARKDOWN-SPECIFICS.md#note-warning-tip-important) for examples.